README.md

```diff
@@ -18,26 +18,59 @@ These instructions will get you a copy of the project up and running on your loc
 
 https://help.github.com/articles/cloning-a-repository/
 
-Open the solution file (.sln) using perfered IDE.
+Open the solution file (.sln) using preferred IDE. Visual Studio Community is recommended.
+
+In order to conduct database functions, you'll need to configure the User Secrets using the database credentials. To do so:
+* Open the .sln in Visual Studio Community
+* In the Solution Explorer, find the "NSSServices" item, right click on it and select "Manage User Secrets".
+* In the secrets.json, copy the following code and fill in the corresponding information for the StreamStats RDS Test instance (found in the WIM Keepass or ask a WIM team member for the information):
+```
+{
+  "dbuser": "",
+  "dbpassword": "",
+  "dbHost": ""
+}
+```
 
 ## Building and testing
 
-No testing files are currently available for this repository
+To run the services locally:
+* Open the .sln file in Visual Studio Community
+* In the Solution Explorer, find the "NSSServices" item, right click on it and select "Settings".
```

Comment:
I had to select "Properties" as I didn't have an option for "Settings" that I saw.
![image](https://user-images.githubusercontent.com/59063181/127338710-42c212a6-ef47-421e-b5b7-4a6daf6260dc.png)